src/plugins/mobilemenu/mobilemenu.css

```diff
@@ -0,0 +1,22 @@
+.mobilemenu{
+  position: absolute !important;
```

Comment:
The use of `!important` should be very limitied as it is already an escalation mechanism.
Probably it's better to make the CSS rule more specific to make it win over the other rules. This would allow the use of `!important` for those cases where it really is important, like in a custom.css config.